projects/1.16.1/assets/extreme-reactors/bigreactors/lang/zh_cn.json

```
@@ -0,0 +1,549 @@
+    "gui.bigreactors.turbine.controller.coolantbar.line8": "等待流体端口抽出",
+    "gui.bigreactors.turbine.controller.vaporbar.line1": "蒸汽缓存槽",
+    "gui.bigreactors.turbine.controller.vaporbar.line2": "  @0",
+    "gui.bigreactors.turbine.controller.vaporbar.line3a": "  @1",
+    "gui.bigreactors.turbine.controller.vaporbar.line3b": " / @2",
+    "gui.bigreactors.turbine.controller.vaporbar.line4a": "  @3",
+    "gui.bigreactors.turbine.controller.vaporbar.line4b": "% 已满",
+    "gui.bigreactors.turbine.controller.vaporbar.line5": "从流体端口输入的蒸汽",
+    "gui.bigreactors.turbine.controller.vaporbar.line6": "会被存储在该缓存槽中",
+    "gui.bigreactors.turbine.controller.vaporbar.line7": "之后用于驱动涡轮转子",
+
+    "gui.bigreactors.turbine.controller.rpmbar.line1": "转子转速",
+    "gui.bigreactors.turbine.controller.rpmbar.line2": "  @0",
+    "gui.bigreactors.turbine.controller.rpmbar.line3": "转子每分钟的",
+    "gui.bigreactors.turbine.controller.rpmbar.line4": "转速（RPM）",
+    "gui.bigreactors.turbine.controller.rpmbar.line5": "转子在 900 或 1800",
+    "gui.bigreactors.turbine.controller.rpmbar.line6": "RPM 时性能最佳",
+    "gui.bigreactors.turbine.controller.rpmbar.line7": "转速超过 2000 RPM 时可能会",
+    "gui.bigreactors.turbine.controller.rpmbar.line8": "导致涡轮机发生灾难性的故障",
+
+    "gui.bigreactors.turbine.controller.rotorstatus.line1": "转子效率",
+    "gui.bigreactors.turbine.controller.rotorstatus.line2": "  @0",
+    "gui.bigreactors.turbine.controller.rotorstatus.line3": "转子叶片蒸汽容量：%1$d mB",
+    "gui.bigreactors.turbine.controller.rotorstatus.line4": "转子叶片：@1",
+    "gui.bigreactors.turbine.controller.rotorstatus.line5": "只有当蒸汽数量大于转子叶片",
+    "gui.bigreactors.turbine.controller.rotorstatus.line6": "蒸汽容量时，转子叶片才能从",
+    "gui.bigreactors.turbine.controller.rotorstatus.line7": "蒸汽中捕获能量",
+    "gui.bigreactors.turbine.controller.rotorstatus.line8": "如果输入的蒸汽超过容量上限",
+    "gui.bigreactors.turbine.controller.rotorstatus.line9": "时效率则会下降",
+    "gui.bigreactors.turbine.controller.rotorstatus.100": "100%",
+
+    "gui.bigreactors.turbine.controller.maxintake.label": "流速：",
+    "gui.bigreactors.turbine.controlrod.maxintake.input.tooltip.line1": "流速调节",
+    "gui.bigreactors.turbine.controlrod.maxintake.input.tooltip.line2": "控制从蒸汽缓存",
+    "gui.bigreactors.turbine.controlrod.maxintake.input.tooltip.line3": "槽中抽取蒸汽的",
+    "gui.bigreactors.turbine.controlrod.maxintake.input.tooltip.line4": "最大速率，可以",
+    "gui.bigreactors.turbine.controlrod.maxintake.input.tooltip.line5": "高效地设置蒸汽",
+    "gui.bigreactors.turbine.controlrod.maxintake.input.tooltip.line6": "处理的最大速率",
+    "gui.bigreactors.turbine.controlrod.maxintake.input.tooltip.line7": "通过点击向上或向下",
+    "gui.bigreactors.turbine.controlrod.maxintake.input.tooltip.line8": "按钮来改变 1 点数值",
+    "gui.bigreactors.turbine.controlrod.maxintake.input.tooltip.line9": "如果按住 Control 键",
+    "gui.bigreactors.turbine.controlrod.maxintake.input.tooltip.line10": "或 Command 键再点",
+    "gui.bigreactors.turbine.controlrod.maxintake.input.tooltip.line11": "击将改变 10 点数值",
```

|Feedback Priority|
|---|
|🌫 Dust / Non-blocking, “take it or leave it”|

```suggestion
    "gui.bigreactors.turbine.controlrod.maxintake.input.tooltip.line10": "或 Command 键再单击",
    "gui.bigreactors.turbine.controlrod.maxintake.input.tooltip.line11": "将改变 10 点数值",
```